tests/functional/pod_and_daemons/test_cephtoolbox_pod_nodeaffinity.py

```diff
@@ -0,0 +1,114 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import ocp, node, constants
+from ocs_ci.ocs.resources import pod
+from ocs_ci.framework.pytest_customization.marks import bugzilla, magenta_squad
+from ocs_ci.framework.testlib import tier1
+from ocs_ci.helpers.sanity_helpers import Sanity
+from ocs_ci.ocs.node import (
+    unschedule_nodes,
+    drain_nodes,
+    schedule_nodes,
+)
+from ocs_ci.ocs.resources.pod import (
+    wait_for_pods_to_be_running,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@magenta_squad
+@bugzilla("2249640")
+class TestCephtoolboxPod:
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+
+        """
+        self.sanity_helpers = Sanity()
+
+    @pytest.fixture(autouse=True)
```

Comment:
Do you want to run teardown for each test case? if not you can use `@pytest.fixture(scope="session", autouse=True)`

Reply:
done.